vmware_aria_operations_integration_sdk/describe.py

```diff
@@ -380,15 +380,15 @@ def add_identifier(
         },
         nsmap=ns_map,
     )
-    add_enum_values(identifier_xml, identifier_json, schema_version, names)
+    add_enum_values(identifier_xml, identifier_json, names, schema_version)
     return identifier_xml
 
 
 def add_enum_values(
-    parent: Element, identifier_json: Dict, schema_version: str, names: _Names
+    parent: Element, identifier_json: Dict, names: _Names, schema_version: int
 ) -> None:
     if "enum_values" in identifier_json:
-        if schema_version == "1.0.0":
+        if schema_version == 1:
```

Comment:
For future compatibility, this should be `>= 1`

Reply:
If we make another change to enums in the schema, then we can modify then, but if we make a change to an unrelated part of the schema, then we don't have to go through all the existing places where we're checking the schema and modify those too.